docker/public/Dockerfile-h2o-release

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:11
+
+RUN mkdir -p /opt/h2oai/h2o-3/
+COPY h2o.jar /opt/h2oai/h2o-3/
+
+CMD java -XX:+UseContainerSupport -jar /opt/h2oai/h2o-3/h2o.jar
```

Comment:
Default, overridable. `-XX:+UseContainerSupport` should be enabled by default according to https://hub.docker.com/_/openjdk , yet I added it just to be sure.